python/sdk/test/integration_test.py

```diff
@@ -1124,3 +1124,42 @@ def test_redeploy_model(integration_test_url, project_name, use_google_oauth, re
 
 def deployment_mode_suffix(deployment_mode: DeploymentMode):
     return deployment_mode.value.lower()[0:1]
+
+
+@pytest.mark.integration
+def test_standard_transformer_simulate(integration_test_url, use_google_oauth):
+    """
+    Validate that set traffic is working when switching from different deployment mode
+    """
```

Comment:
Please update or remove this comment as this is not related to the test implementation.